libs/langchain/langchain/llms/promptguard.py

```diff
@@ -0,0 +1,105 @@
+import logging
+from typing import Any, Dict, List, Optional
+
+from pydantic import Extra, root_validator
+
+from langchain.callbacks.manager import CallbackManagerForLLMRun
+from langchain.llms.base import LLM
+from langchain.prompts.base import StringPromptValue
+from langchain.utils import get_from_dict_or_env
+
+logger = logging.getLogger(__name__)
```

Comment:
Does LangChain use this as well?

Reply:
the `env` config or the `logger`? Yes they use both.